src/VisualStudio/Core/Def/Implementation/AnalyzerDependency/AnalyzerDependencyCheckingService.cs

```diff
@@ -80,48 +80,57 @@ internal sealed class AnalyzerDependencyCheckingService
 
             var conflicts = results.Conflicts;
             var missingDependencies = results.MissingDependencies;
+            var projects = _workspace.DeferredState.ProjectTracker.ImmutableProjects;
```

Comment:
any reason this is explicitly set to this? @tmeschter 
 results = await GetConflictsAsync().ConfigureAwait(continueOnCapturedContext: **true**); 

if that is flipped to false then I think rest of the code would have been running BG.